### DISCUSSION TITLE
"Internal error" "kvEventSink is not enabled"

### DISCUSSION THREAD

**Discussion by sdjeong-zenaad** [tendermint/tendermint#8742] [Q&A] [answered]
I was following this tutorial (https://docs.tendermint.com/master/introduction/quick-start.html#local-node)
However, a error occurs at the 'Sending Transaction'.
```sh
$ curl -s 'localhost:26657/broadcast_tx_commit?tx="abcd"'
{"code":-32603,"message":"Internal error","data":"cannot confirm transaction because kvEventSink is not enabled"}
```
Why do I have this error?

**Comment by creachadair** [comment, marked as answer]
Hi, @sdjeong-zenaad. This error means that the transaction indexer is not enabled. The transaction should still have been posted successfully.

Since the code on the `master` branch is unreleased, I recommend using the v0.35 release instead: https://docs.tendermint.com/v0.35/introduction/quick-start.html

Alternatively, you can enable the indexer in the `[tx-index]` section of the `config.toml` file.

**Reply by sdjeong-zenaad** [reply]
Thank you.

**Comment by HiccopABC** [comment]
May I ask if you solved it? How was it solved?I have the same problem

**Comment by sergio-mena** [comment]
@HiccopABC feel free to ask the question here https://github.com/cometbft/cometbft/
